thirdparty.js

```diff
@@ -203,9 +203,10 @@ Thirdparty.fromKryptoKit = function (entropy, password) {
     /* eslint-disable new-cap */
     var decipher = new aesjs.ModeOfOperation.ecb(aesKey)
     /* eslint-enable new-cap */
+    /* decrypt returns an Uint8Array, perhaps there is a better way to concatenate */
     privKey = Buffer.concat([
-      decipher.decrypt(encryptedSeed.slice(0, 16)),
-      decipher.decrypt(encryptedSeed.slice(16, 32))
+      Buffer.from(decipher.decrypt(encryptedSeed.slice(0, 16))),
+      Buffer.from(decipher.decrypt(encryptedSeed.slice(16, 32)))
```

Comment:
Tested this locally/manually, resulted in the same values with and without ``Buffer``, so I'm not completely getting the reason here, assume this is an extra security measure. Anyway, also makes things a bit more explicit and minimally doesn't hurt.

Reply:
It is a `Uint8Array` which cannot be implicitly used in `Buffer.concat`, seemingly.